materials/r-sql.md

```diff
@@ -18,42 +24,46 @@ portalDB <- "portal_mammals.sqlite"
 conn <- dbConnect(SQLite(), portalDB)
 ```
 
-## Checkout the database
+### Check out database structure
 
 ```
 dbListTables(conn)
 dbListFields(conn, "plots")
 dbListFields(conn, "surveys")
 ```
 
-## Run queries
+### Run queries
 
 ```
 query <- "SELECT genus, species, COUNT(*)
           FROM surveys JOIN species
-		  ON surveys.species_ID = species.species_ID
-		  GROUP BY genus, species"
+          ON surveys.species_ID = species.species_ID
+          GROUP BY genus, species;"
 species_counts <- dbGetQuery(conn, query)
 ```
 
-**Exercise 1**
+> Do [Exercise 1 - Connect and Query]({{ site.baseurl }}/exercises/R-sql-connect-and-query-R).
+
 
+### Write new information to database
 
-## Write new information to database
+* Write as a table
 
 ```
 dbWriteTable(conn, "SpeciesCounts", species_counts)
+dbListTables(conn)
 ```
 
-* Show that this table now exists in the database
```

Comment:
I think it would be good to keep this using the new `> ` format in the same way that you did for showing them the view. I like showing this immediately before doing running and showing the view change.